examples/oas_testcase_openapi.yaml

```diff
@@ -1,2 +1,12 @@
-null
-...
+components_schemas: {}
```

Comment:
Oh, now that I see this sample file, I wonder why have it duplicated around instead of just reading this :-)